consensus/istanbul/backend/engine.go

```diff
@@ -285,6 +286,57 @@ func (sb *backend) VerifyHeaders(chain consensus.ChainReader, headers []*types.H
 	return abort, results
 }
 
+func (sb *backend) disconnectReleasedVal(header *types.Header, curVals istanbul.ValidatorSet) error {
+	if curVals == nil {
+		return errors.New("validator set is nil")
+	}
+	if header == nil {
+		return errors.New("header is nil")
+	}
+	// Istanbul BFT chain does not have nil for `sb.broadcaster`,
+	// This addition is for existing testbase that did not correctly intiialize
+	// braodcaster. Theferoe, it can be interpreted that the network setup
```

Comment:
```suggestion
	// This addition is for existing testbase that did not correctly initialize
	// broadcaster. Therefore, it can be interpreted that the network setup
```